terraform/benchmark/README.md

```diff
@@ -0,0 +1,67 @@
+# Overview
+
+This folder contains an executable description of a benchmarking service that 
+compares the performance of Forest relative to Lotus. The benchmark results are 
+uploaded to an S3 bucket, where they are stored in both weekly file and `all_result` file.
```

Comment:
Which versions of Forest against which versions of Lotus? Latest releases? This should be specified in the README. Are the releases updated automatically?

Reply:
corrected